gradle/publish-dists.gradle

```diff
@@ -1,7 +1,18 @@
 apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.artifactory'
 apply plugin: 'com.jfrog.bintray'
 
-//Bintray publish
+// If you modify bintray or artifactory publishing you also need to modify publish-jars.gradle
+
+publishing {
+  publications {
+    dists(MavenPublication) {
```

Comment:
Switching from 

```
publishing.publications {
     artifactory(MavenPublication) {
         artifact distTar
         artifactId applicationName
         groupId 'com.palantir.atlasdb'
     }
}
```

does not add `-dist` classifier. The name -s now ``atlasdb-timelock-version.sls.tgz`` instead of ``atlasdb-timelock-version-dist.sls.tgz``